### PR TITLE
[jansson] Update to 2.13.1

### PR DESCRIPTION
**Pull request by Jimmy-Hu**
**Describe the pull request**

- What does your PR fix? Update jansson package to 2.13.1

  https://github.com/akheron/jansson/releases/tag/v2.13.1

### CODE REVIEW

File: ports/jansson/portfile.cmake [hunk 1]
```diff
@@ -3,8 +3,8 @@ include(vcpkg_common_functions)
 vcpkg_from_github(
   OUT_SOURCE_PATH SOURCE_PATH
   REPO akheron/jansson
-  REF 71c4e8ec215afa225ac20eed269a14963cd37b50
-  SHA512 cdb955996768d6c7ed15b9f1bb7ddf4905f881c4e604d9e7a863f42c513eaaa9fb8799dacfa392424fbf725aac125d4716e10c44c3415449b5c5edd38a87b290
+  REF e9ebfa7e77a6bee77df44e096b100e7131044059
+  SHA512 88a59c1cf5150699def17c86192ca9bacdfe6669319f770c3fbf14fa8edc48b4bb015a4a634a09db40fba9054320ac7c133c4d156f813af540a636f7825f0610
   HEAD_REF master
 )
```
Comment (NancyLi1013) [inline ports/jansson/portfile.cmake:10]: Could you please also update 

`file(INSTALL ${SOURCE_PATH}/LICENSE DESTINATION ${CURRENT_PACKAGES_DIR}/share/jansson RENAME copyright)`

as

`file(INSTALL ${SOURCE_PATH}/LICENSE DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT} RENAME copyright)`?

**Review by NancyLi1013 — CHANGES_REQUESTED** [review]

**Comment by NancyLi1013** [comment]
Hi @Jimmy-Hu 
Thanks for this PR.
Could you please resolve the conflicts and also address the review suggestions?

**Comment by Jimmy-Hu** [comment]
> 
> 
> Hi @Jimmy-Hu
> Thanks for this PR.
> Could you please resolve the conflicts and also address the review suggestions?

@NancyLi1013 Thank you for your reply. Let me check and update that.

**Review by NancyLi1013 — APPROVED** [review]

**Comment by NancyLi1013** [comment]
LGTM now. Thanks for your efforts.

**Pull request merged**